src/components/BooksContainer/BookList/Book.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import '../../styles/book.css';
+
+const temp = [];
+
+const Book = () => (
+  temp.map((el) => (
+    <div key={el.id} className="book-container d-flex">
+      <div className="general-info d-flex">
+        <span className="genre">{el.genre}</span>
+        <h3 className="title">{el.title}</h3>
+        <span className="author">{el.author}</span>
+        <ul className="book-management d-flex">
+          <li>Comments</li>
+          <li>Remove</li>
```

Comment:
- [ ] This should be a button since it will have the function of deleting books. So, best you enclose it within button tags

Reply:
@mke2111 Since It wasn't rendered for now I forgot to make them as buttons. I will do that right away